portal/src/error/duplicated.ts

```diff
@@ -0,0 +1,4 @@
+export interface APIDuplicatedIdentityError {
+  errorName: string;
+  reason: "DuplicatedIdentity";
```

Comment:
This error is going to be removed in https://github.com/authgear/authgear-server/pull/488 https://github.com/authgear/authgear-server/pull/488/commits/a14e7eb09ca0578e438d3a418afcc2524423f24c

Reply:
It's already removed, will delete this file